docker/compose/validator-testnet/.gitattributes

```diff
@@ -0,0 +1,3 @@
+genesis.blob -whitespace
+libra_root_key -whitespace
+waypoint.txt -whitespace
```

Comment:
Given the other PR i put out you should remove these I think

Reply:
Also why do we need to checkin empty files?

Reply:
the way the bind mount for docker works is that if these do not exist, the container won't start. the user would then need to create them... I'm probably being too nice :P

Reply:
I mean if these don't exist maybe you don't want docker to start? idk :D

Reply:
🤡